site/content/how-to/monitoring/prometheus.md

```diff
@@ -11,16 +11,96 @@ docs: "DOCS-1418"
 ## Overview
 
 
-NGINX Gateway Fabric metrics are displayed in [Prometheus](https://prometheus.io/) format, simplifying monitoring. You can track NGINX and controller-runtime metrics through a metrics server orchestrated by the controller-runtime package. These metrics are enabled by default and can be accessed on HTTP port `9113`.
-
+NGINX Gateway Fabric metrics are displayed in [Prometheus](https://prometheus.io/) format. These metrics are served through a metrics server orchestrated by the controller-runtime package on HTTP port `9113`. When installed, Prometheus automatically scrapes this port and collects metrics. [Grafana](https://grafana.com/) can be used for rich visualization of these metrics.
 
 {{<call-out "important" "Security note for metrics">}}
 Metrics are served over HTTP by default. Enabling HTTPS will secure the metrics endpoint with a self-signed certificate. When using HTTPS, adjust the Prometheus Pod scrape settings by adding the `insecure_skip_verify` flag to handle the self-signed certificate. For further details, refer to the [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#tls_config).
 {{</call-out>}}
 
+## Installing Prometheus and Grafana
+
+{{< note >}}These installations are for demonstration purposes and have not been tuned for a production environment.{{< /note >}}
+
+### Prometheus
+
+```shell
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+helm install prometheus prometheus-community/prometheus -n monitoring --create-namespace --set server.global.scrape_interval=15s
+```
+
+Once running, you can access the Prometheus dashboard by using port-forwarding in the background:
+
+```shell
+kubectl port-forward -n monitoring svc/prometheus-server 9090:80 &
+```
+
+Visit [http://127.0.0.1:9090](http://127.0.0.1:9090) to view the dashboard.
+
+### Grafana
+
+
+```shell
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+helm install grafana grafana/grafana -n monitoring --create-namespace
+```
+
+Once running, you can access the Grafana dashboard by using port-forwarding in the background:
+
+```shell
+kubectl port-forward -n monitoring svc/grafana 3000:80 &
+```
+
+Visit [http://127.0.0.1:3000](http://127.0.0.1:3000) to view the Grafana UI.
+
+The username for login is `admin`. The password can be acquired by running:
+
+```shell
+kubectl get secret -n monitoring grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
+```
+
+#### Configuring Grafana
+
+In the Grafana UI menu, go to `Connections` then `Data sources`. Add your Prometheus service (`http://prometheus-server.monitoring.svc`) as a data source.
+
+Download the following sample dashboard and Import as a new Dashboard in the Grafana UI.
+
+{{< download "static/grafana-dashboard.json" "ngf-grafana-dashboard.json" >}}
```

Comment:
The first parameter here should be just `grafana-dashboard.json` without `static/`.
See https://gohugo.io/content-management/static-files/ - Hugo seems to map anything in `/static` to the root of the server.

Reply:
I've added a fix for that on the hugo theme side. I'll update your theme version on the netlify preview and try it.

Reply:
@sjberman https://deploy-preview-1620--nginx-gateway-fabric.netlify.app/nginx-gateway-fabric/how-to/monitoring/prometheus/ is updated now, I think it's looking good now 👍

Reply:
awesome, thank you! Can this hugo update be merged in and updated in our repo? I don't want to merge this PR and have the docs rendering fail for anyone else because we aren't yet using the latest version.

Reply:
Yes, I'll get the theme change approved, merged, and updated. Will comment here when it's good to merge.

Reply:
The theme has been updated, you should be good to merge.

Reply:
@nginx-jack Thanks! Do I need to update the hugo theme in this repo as part of this PR?

Reply:
Yeah, it's worth updating `site/go.mod` for local development.
The latest is;
`require github.com/nginxinc/nginx-hugo-theme v0.41.0`